### ISSUE TITLE
App crashed when I clicked on app back icon in settings 

### ISSUE THREAD

**Issue by darewealth** [JavaCafe01/PdfViewer#33]
### Issue Summary

When I clicked on the app back icon in settings after I open a pdf file the app crashed.

### Steps to Reproduce the Problem
* Click on + icon on the app 
* Click on pick file to select any pdf file
* Open a pdf file and click the 3 dots at the top 
* Then click on settings and click app back icon 
* And note the bug.

### Screen recording.
https://youtu.be/OSfT5J5OfC4

### Specifications

  - App Version:3.1
  - Android Version:7.0
  - Device: Gionee X1

### Log 
```07-08 22:41:03.837 19475 19475 E AndroidRuntime: FATAL EXCEPTION: main
07-08 22:41:03.837 19475 19475 E AndroidRuntime: Process: com.gsnathan.pdfviewer, PID: 19475
07-08 22:41:03.837 19475 19475 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.gsnathan.pdfviewer/com.gsnathan.pdfviewer.MainActivity_}: java.lang.SecurityException: Permission Denial: reading com.android.externalstorage.ExternalStorageProvider uri content://com.android.externalstorage.documents/document/primary%3AXender%2Fother%2Fmaxadvance-form-private-organizations.pdf from pid=19475, uid=10125 requires android.permission.MANAGE_DOCUMENTS, or grantUriPermission()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2417)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2477)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.app.ActivityThread.-wrap11(ActivityThread.java)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1345)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:148)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5452)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:781)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:671)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: Caused by: java.lang.SecurityException: Permission Denial: reading com.android.externalstorage.ExternalStorageProvider uri content://com.android.externalstorage.documents/document/primary%3AXender%2Fother%2Fmaxadvance-form-private-organizations.pdf from pid=19475, uid=10125 requires android.permission.MANAGE_DOCUMENTS, or grantUriPermission()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:1599)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:235)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:176)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.content.ContentProviderProxy.query(ContentProviderNative.java:421)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.content.ContentResolver.query(ContentResolver.java:491)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.content.ContentResolver.query(ContentResolver.java:434)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at com.gsnathan.pdfviewer.e.b()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at com.gsnathan.pdfviewer.e.a()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at com.gsnathan.pdfviewer.e.e()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at com.gsnathan.pdfviewer.MainActivity_.a()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at f.a.b.a.c.a()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at com.gsnathan.pdfviewer.MainActivity_.setContentView()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at com.gsnathan.pdfviewer.MainActivity_.onCreate()
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:6305)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1110)
07-08 22:41:03.837 19475 19475 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2370)```

**Comment by kofaysi** [comment]
The same on /e/ OS https://gitlab.e.foundation/e/backlog/-/issues/1667

**Comment by JavaCafe01** [comment]
Fixed in PR #72
